### PR TITLE
Removing fastapi constraints + upgrade

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?

- ⬆️ Removing fastapi version constrains
- ♻️ Fixing custom exception handler (it should not be initialized in the startup even handler function)

## Related issue/s

- fixes https://github.com/ITISFoundation/osparc-simcore/issues/3896

### CODE REVIEW

File: packages/service-library/src/servicelib/fastapi/long_running_tasks/_server.py [hunk 1]
```diff
@@ -40,13 +40,14 @@ async def on_startup() -> None:
...
         if app.state.long_running_task_manager:
             task_manager: TasksManager = app.state.long_running_task_manager
             await task_manager.close()
 
     app.add_event_handler("startup", on_startup)
     app.add_event_handler("shutdown", on_shutdown)
+
+    # add error handlers
+    # NOTE: Exception handler can not be added during the on_startup script, otherwise not working correctly
+    app.add_exception_handler(BaseLongRunningError, base_long_running_error_handler)
```
Comment (sanderegg) [inline packages/service-library/src/servicelib/fastapi/long_running_tasks/_server.py:53]: 👍 and I get it you checked we did it correctly in the services right?

**Review by sanderegg — APPROVED** [review]
Very cool! thanks for checking this nasty issue.

File: packages/service-library/src/servicelib/fastapi/long_running_tasks/_server.py (hunk 1 above)
Comment (matusdrobuliak66) [reply packages/service-library/src/servicelib/fastapi/long_running_tasks/_server.py:53]: Yes correct 👍 this was the only place we have it in the startup event function

**Review by pcrespov — APPROVED** [review]
:tada: thx so much!

**Pull request merged**